packages/protocol/contracts/team/airdrop/MockAddressManager.sol

```diff
@@ -0,0 +1,28 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.24;
+
+/// @title Mock Address Manager
+/// @notice Provides a simplified mock-up of an address manager for testing purposes,
+/// substituting for a more complex AddressManager in test environments.
+/// @dev This mock manager is used in tests to simulate interactions with a vault system
+/// that typically involves managing addresses of various contracts like an ERC20Vault.
+contract MockAddressManager {
```

Comment:
Could you move these 2 (Mock + SimpleERC20Vault) contracts to a different location ? As none of them would be used production, and also in prod the SimpleERC20Vault will be a multisig. Better moving to `/test` or `/helper` ?